tests/cases/compiler/jsFileMethodsOverloads.ts

```diff
@@ -0,0 +1,50 @@
+// @allowJs: true
+// @noEmit: true
```

Comment:
please use `// @outDir: dist/` and `// @declaration: true` to make sure that js->d.ts conversion works too.

Reply:
@sandersn I tried this but didn't see any changes in the baseline files. What outcome I should be observing?

Reply:
Did you remove `@noEmit: true`. You should see a baseline named jsFileMethodsOverloads.js and its last section should be a .d.ts file.

Reply:
@sandersn Yup, that was it. Seems like it's working fine now. Thank you!